extract.c

```diff
@@ -109,7 +111,7 @@ char revcomp(char b) {
         case 'T':
         case 't':
             return 'A';
-        default:
+
```

Comment:
Any reason you removed the `default:`?

Reply:
This appears to have happened by accident, will re-add.